ranch/ranch.go

```diff
@@ -124,6 +124,7 @@ func (r *Ranch) Acquire(rType, state, dest, owner, requestID string) (*crds.Reso
 	})
 
 	var returnRes *crds.ResourceObject
+	var createdTime time.Time
```

Comment:
this isn't set if a `requestID` isn't set right? which I think means that requests without a request ID will have a very long duration reported as a result.

Reply:
might want a test case to cover this, too.

Reply:
Updated.

Reply:
I think this still isn't resolved - it doesn't look like `createdTime` will be set to anything if `requestID` is empty. (`requestID` is an optional parameter, I believe.)

This would result in erroneously large durations being reported.

Reply:
maybe initialize this to `r.now()`? it'd give reasonable results for requests without a request ID (basically just the RTT to make Kubernetes API requests). would be easy to test this functionality, too.

Reply:
Updated.